dspot-web/README.md

```diff
@@ -0,0 +1,33 @@
+# Introduction
+
+This is a prototype web interface of Dspot, inspired from [CommitGuru](http://commit.guru/).
+
+Author: Henry Luong
+
+## Screenshots
+
+TODO add screenshots and explanation.
+
+## Running
+
+```
+cd dspot-web
+npm install --no-optional
+npm run-script start
+```
+
+## Screenshots
+
+![Pic 1](https://github.com/Tailp/dspot/blob/web/dspot-web/screenshots/pic1.png)
```

Comment:
Thanks for the screenshots. Could you add one paragraph per screenshot to explain what they show?

Reply:
merge pic1 and pic2 they are the same.